In [ ]:
# установить свежую версию, чтобы использовать ф-ю wilcoxon
!pip install scipy==1.5.2

In [ ]:
import math
import numpy as np
import pandas as pd
from scipy import stats as st
import matplotlib.pyplot as plt 

В данном модуле будут рассмотрены критерии для проверки гипотезы об отсутсвии сдвига. 

Критерия называется непараметрическим, если закон распределения статистики критерия, независит от закона распределения исходной генеральной совокупности.

Рассмотрим две генеральные совокупности $\xi$ и $\eta$. Будем предполагать, что две сл. вел. $\xi$ и $\eta$ независимы.

Пусть ген. совокупность $\xi$ имеет ф-ю распределения $F(x)$ и из данной ген. совокупности получена выборка $X = (x_1, ..., x_n)$

$\xi, F(x), X = (x_1, ..., x_n)$

$\eta, G(x), Y = (y_1, ..., y_n)$

Требуется проверить гипотезу о том, что ф-и распределения совпадают.

$H_0: F(x) = G(x)$ для всех $x \in \mathbb {R}$

$H_1: F(x) \geq G(x)$ для всех $x \in \mathbb {R}$

$H'_1: F(x) \leq G(x)$ для всех $x \in \mathbb {R}$

$H''_2: F(x) \neq G(x)$ для всех $x \in \mathbb {R}$

Будем предполагать, что ф-и распр. $F(x)$ и $G(x)$ являются непрерывными, откуда следует, что эл-ты из выборок $X, Y$ не должны повторятся.

#### Непараметрические критерии для одномерных выборок

##### Критерий однородности Вилкоксона

Будем предполагать, что объем выборки $m$ из ген. совокупности $\eta$ не превосходит объема выборки $n$ из ген совокупности $\xi$.

Пусть $m \leq n$

Составим общий вариационный ряд из эл-ов выборки $X, Y$.

$ Z_{[n+m]} = (X_{[n]}, Y_{[m]}) $

$z_{(1)} < z_{(2)} < ... < z_{(m+n)}$

Назовём номер позиции эл-та выборки в вариационном ряду рангом. Найдём ранги элементов выборки $Y$. Через $s_1$ обозначим ранг эл-та $y_1$

$rank(y_1) = s_1$

Рассмотрим вычисления ранга на простом примере

$X_{[4]} = (8, 4, 6, 1)$

$Y_{[3]} = (0, 2, 3)$

Составим общий вариационный ряд, а под ним укажем ранг.

$0, 1, 2, 3, 4, 6, 8$

$1, 2, 3, 4, 5, 6, 7$

$rank(y_1) = 1$, $rank(y_2) = 3$, $rank(y_3) = 4$

$s_1 = 1$, $s_2 = 3$, $s_3 = 4$

Если сл. величины имеют одинаковые ф-ии распределения, в этом случае эл-ты выборок $X, Y$ будут равномерно перемешаны в вариационном ряду для $z$. 

Но если  $F(x) \geq G(x)$, тогда эл-ты выборки $X$ будут принимать значения меньшие, чем эл-ты выборки $Y$ и следовательно эл-ты выборки $Y$ будут сосредоточены в правой части вариационного ряда и ранги их будут больше.

Если $F(x) \leq G(x)$ эл-ты выборки $Y$ будут принимать меньшие значения, чем эл-ты выборки $X$ будут сосредоточены в левой части вариационного ряда для $z$ и ранги их будут в среднем меньше, чем ранги иксов.

###### Статистический критерий

В качестве статистики критерия, можно выбрать сумму рангов эл-ов выборки из генеральной совокупности $\eta$ (из выборки $Y$).

$W = \sum \limits_{i=1}^m s_i$

данная статистика принимает значения в интервале 

$[m (m+1)/2, mn + (m+1)m/2]$

###### Критическая область

Обратим внимание, что интервалы включают в себя крайние точки.

$H_1: F(x) \leq G(x)$

$V_k: [c_1, mn + (m+1)m/2]$, где

$c_1$ - некоторая критическая точка

$H_1: F(x) \geq G(x)$

$V_k: [m (m+1)/2, c_2]$

$H_1: F(x) \neq G(x)$

$V_k: [m (m+1)/2, c_3] \cup [c_4, mn + (m+1)m/2]$

Критические значения $c_1, c_2, c_3, c_4$ находят по таблицам критических значений статистики Вилкоксона или с помощью стат. пакетов, из условия

$P(W \in V_k|H_0) = \alpha$

если $W \in V_k \Rightarrow H_0$ отклоняется!

###### Задача №1

Кокаин чрезвычайно вреден для сердца, он может вызвать инфаркт миокарда даже у молодых людей без атеросклероза. Кокаин сужает коронарные сосуды, что приводит к уменьшению притока крови к миокарду кроме того, он ухудшает насосную функцию сердца. Нифедипин (препарат из группы антагонистов кальция) обладает способностью расширять сосуды, его применяют при ишемической болезни сердца. Ш. Хейл и соавт. (S. L. Hale, К. J. Alker, S. H. Rezkalla et al. Nifedipine protects the heart from the acute deleterious effects of cocaine if administered before but not after cocaine. Circulation, 83:1437—1443, 1991) предположили, что нифедипин можно использовать и при поражении сердца, вызванном кокаином. Собакам вводили кокаин, а затем нифедипин либо физиологический раствор. Показателем насосной функции сердца служило среднее артериальное давление. Были получены следующие данные.

Среднее артериальное давление после приема кокаина, мм рт. ст.

Плацебо
156, 171, 133, 102, 129, 150, 120, 110, 112, 130

Нифедипин
73, 81, 103, 88, 131, 106, 107, 111, 122, 108

Влияет ли нифедипин на среднее артериальное давление после приема кокаина?

Постройте статистику критерия, приняв за $Y_i$
​	
  наблюдения артериального давления при приеме Нифедипина

*С.Гланц Медико-биологическая статистика

**Решение:**

$\xi$- плацебо, выборка $X$, ф-я распр. $F(x)$

$\eta$ - нифедипин, выборка $Y$, ф-я распр. $G(x)$

Проверяем гипотезу

$H_0: F(x) = G(x)$

$H_1: F(x) \neq G(x)$


В качестве статистики критерия возьмем 

$W = \sum \limits_{i=1}^m s_i$, где

$s_i$ - ранги эл-ов выборки из генеральной совокупности $\eta$ (из выборки $Y$).

In [ ]:
X = np.array([156, 171, 133, 102, 129, 150, 120, 110, 112, 130])
Y = np.array([73, 81, 103, 88, 131, 106, 107, 111, 122, 108])
concatenate = np.concatenate((X, Y))
Z = np.sort(concatenate)
# в массиве Z, ищим эл-ты из Y
mask = np.in1d(Z, Y)
# получим индексы = ранг
# получим ранг нумерация с 0, нам нужно с 1, поэтому добавим всем рангам единицу
s = np.where(mask)[0] + 1
# статистика Вилкоксона
W = np.sum(s)
W

71

Влияет ли нифедипин на среднее артериальное давление после приема кокаина?

Найдите **область принятия гипотезы**, приняв за $Y_i$ наблюдения артериального давления при приеме Нифедипина. Примите статистическое решение. Уровень значимости 0.05

Полезные статьи: [студопедия](https://studopedia.ru/9_222473_kriteriy-vilkoksona.html), [блог](https://baguzin.ru/wp/neparametricheskie-kriterii-rangovy/)

Статистика критерия $W \in [m (m+1)/2, mn + (m+1)m/2]$

при $H_0$ распределение суммы рангов $W = \sum s_i$ при достаточно больших объемах выборок $( n, m > 8 )$, стремиться к нормальному c мат.ожиданием 

$EW = m*(m+1)/2+m*n/2$ в середине отрезка

и диспресией

$DW=n∗m∗(m+n+1)/12$

In [ ]:
alpha = 0.05 # уровень значимости
n, m = len(X), len(Y) # объемы выборок
mt = n*(n+m+1)/2 # мат. ожидание W при H0
se = np.sqrt(m*n*(m+n+1)/12)# дисперсия W при H0
zkp1 = st.norm.ppf(alpha/2)# z - критическое значение (левое)
zkp2 = st.norm.ppf(1-(alpha/2))# z - критичесокое значений (правое)
# zkp2 = (n + m + 1) * n - wkp1 # альтернативно, можно вычислить по этой формуле https://studopedia.ru/9_222473_kriteriy-vilkoksona.html
Wdown = mt + zkp1*se
Wup = mt + zkp2*se
print(Wdown, Wup)

79.07211359131887 130.92788640868113


$V_k: (-\infty, 79) \cup (130, +\infty)$

$W \in V_k \Rightarrow H_0$ отвергается. Это означает, что выборки до и после приёма препарат различаются, значит лекарство действует.

**Ответ:**

$W = 71$

$V_k: (-\infty, 79) \cup (130, +\infty)$

$W \in V_k \Rightarrow H_0$ отвергается

###### Апроксимация

В случае, когда объемы выборок велики, мы можем воспользоваться следующей апроксмацией, мы можем центрировать и нормировать статистику $W$

$ \tilde W = \frac{W - \frac{m(m+n+1)}{2}}{\sqrt{ \frac{mn(m+n+1)}{12} }} \xrightarrow[m,n \rightarrow \infty]{H_0 \; d} N(0, 1)$

Для больших объемов выборок, мы можем построить приближенный критерий

$H_0: F(x) = G(x)$

$H_1: F(x) \neq G(x)$

нам достаточно найти квантиль стандартного нормального распределения

$u_{1 - \frac{\alpha}{2}}$

и посмтроить критическую область

$V_k = (|\tilde w_в| > u_{1 - \frac{\alpha}{2}})$

$\tilde w_в \in V_k \Rightarrow H_0$ отклоняем

Если у нас большие выборки, то могут появится одинаковые значения, например, из-за округления. В данном случае необходимо произвести поправку для $\tilde W$ в знаменателе. 

В знаменатель ставим такое выражение

$\sqrt{ \frac{mn(m+n+1)}{12} \bigg (1 - \frac{\sum \limits_{i=1}^k t_i (t_i^2 - 1)}{(n+m)(n+m-1)(n+m+1)} \bigg ) }$

$\tilde W =  \frac{W - \frac{m(m+n+1)}{2}}{\sqrt{ \frac{mn(m+n+1)}{12} \bigg (1 - \frac{\sum \limits_{i=1}^k t_i (t_i^2 - 1)}{(n+m)(n+m-1)(n+m+1)} \bigg ) }}$

$k$ число связок, т.е. наборов повторяющихся значений

$t_i$ - размер $i$-й связки, учитываются только те связки, в которые входят эл-ты из разных выборок. если в связку входят эл-ты из одной и той же выборке, в этом случае они не влияющт на статистику $\tilde W$

##### Критерий Манна-Уитни

###### Статистический критерий

В данном критерии все эл-ты выборки $X$ сравниваются со всеми эл-ами выборки $Y$ и подсчитывается число случаев, когда $x_i < y_j$, т.е. статистика

$U = \sum \limits_{i=1}^n \sum \limits_{i=1}^m I(x_i , y_j)$, где

$I$ - индикатор

т.к. мы предполагаем, что выборки взяты из непрерывного распределения, мы предполагаем, что значения эл-ов выборки не повторяется.

Статистика Манна-Уитни принимает значения из интервала

$U \in (0, mn) $

Взаимосвязь критериев Вилкоксона и Манна-Уитни

$W = U + \frac{m(m+1)}{2}$



###### Критическая область

Мы можем построить критические области таким же образом, как мы строили для статистики Вилкоксона.

Обратим внимание, что интервалы включают в себя крайние точки.

$H_1: F(x) \leq G(x)$

$V_k: [c_1, mn + (m+1)m/2]$, где

$c_1$ - некоторая критическая точка

$H_1: F(x) \geq G(x)$

$V_k: [m (m+1)/2, c_2]$

$H_1: F(x) \neq G(x)$

$V_k: [m (m+1)/2, c_3] \cup [c_4, mn + (m+1)m/2]$

Критические значения $c_1, c_2, c_3, c_4$ находят по таблицам критических значений статистики Манна-Уитни или с помощью стат. пакетов, из условия

$P(W \in V_k|H_0) = \alpha$

если $W \in V_k \Rightarrow H_0$ отклоняется!

Если у нас объемы выборок велики, мы можем воспользоваться ассимптотикой, в этом случае нам необходимо центрировать и нормировать сл. вел. $U$, так же как и Вилкоксона, единственное отличие, отличаются мат. ож.

$EU = \frac{mn}{2}$

$\tilde U = \frac{U - \frac{mn}{2}}{\sqrt{ \frac{mn(m+n+1)}{12} }}$

#### Непараметрические критерии для парных выборок

Сл. вел. зависимы, т.к. измерения производились на одном и том же объекте.

На практике мы прибегаем к данному критерию, в случае, когда два измерения производились на одних и тех же объектах. Например, измерили температуру у одних и тех же людей до и после приема лекарства.

Получена двумерная выборка $(\eta, \xi)$, $(X, Y)$.

Требуется проверить гипотезу о равенстве ф-й распределений, т.е. мы проверяем, есть ли изменения или нет.

$H_0: F(x) = G(x)$ для всех $x \in \mathbb {R}$

$H_1: F(x) \leq G(x)$ для всех $x \in \mathbb {R}$

$H'_1: F(x) \geq G(x)$ для всех $x \in \mathbb {R}$

$H''_1: F(x) \neq G(x)$ для всех $x \in \mathbb {R}$


т.к. наши сл. величины связаны, рассмотрим новую сл. вел. $\zeta$

$\zeta = \eta - \xi$

$z_i = x_i - y_i$

В случае когда справедлива $H_0$ гипотеза, события $(z_i < 0)$ и $(z_i > 0)$ равновероятны, если у нас выборки взяты их непрерывной ген. совокупности, в этом случае данные вероятности будут равны $\frac{1}{2}$.

Поэтому давайте вместо гипотезы $H_0: F(x) = G(x)$ будем рассматривать гипотезу 

$H_0: P(z_i < 0) = P(z_i > 0) = \frac{1}{2}$

$H_1: P(z_i < 0) > P(z_i > 0)$

$H'_1: P(z_i < 0) < P(z_i > 0)$

$H''_1: P(z_i < 0) \neq P(z_i > 0)$

Важно!!! Из равенства $F(x) = G(x) \Rightarrow P(z_i < 0) = P(z_i > 0)$ обратное не верно!

Гипотеза о равенстве вероятности, скорее гипотеза о равенстве медиан двух генеральных совокупностей.

##### Критерий знаков (биноминальный критерий)



###### Критическая область

Определим успех как $z_i < 0$, тогда выборка $Z = (z_1, ..., z_n)$ это последовательность испытаний по схеме Бернулли и в случае справедливости $H_0$ вер-ть появления успеха равна $\frac{1}{2}$

###### Статистический критерий 

$L = \sum I(z_i < 0) \sim Bin(n, \frac{1}{2})$ в случае, когда справедлива $H_0$

Статистика принимает значения из интервала

$L \in [0, n]$

$H_1: F(x) \geq G(x)$

$V_k: (c_1, n]$

$H_1: F(x) \leq G(x)$

$V_k: [0, c_2)$

$H_1: F(x) \neq G(x)$

$V_k: [0, c_3) \cup (c_4, n]$

Критические значения $c_1, c_2, c_3, c_4$ находят по таблицам вероятностей биноминального распределения, в предположении, что справедлива $H_0$ и $P(z_i < 0) = \frac{1}{2}$, для этого нам нужно будет ещё знать объем выборки $n$ или с помощью стат. пакетов, из условия

$P(L \in V_k|H_0) = \alpha$



###### Scipy

Найти $c_i$ можно в пакете $scipy.stats.binom.ppf(alpha, n, p)$, где

$alpha$ - уровень значимости

$n$ - кол-во выборки

$p$ - параметр вер-ти биноминального распределения (p=0.5)

если $L \in V_k \Rightarrow H_0$ отклоняется!

##### Критерий знаковых ранговых сумм Вилкоксона

Статистика основана на рангах модулей $z_i$. Рассмотрим выборку $z_1, ..., z_n$, где $z_i = x_i - y_i$.

На основе её построим последовательность модулей

$|z_1|, ..., |z_n|$

и упорядочим их

$|z'_1| < ... < |z'_n|$

Найдём ранги модулей $|z_i|$ в этой последовательности

$s_1 = rank(|z_1|), ..., s_n=rank(|z_n|)$

т.е. берем модуль $|z_1|$ и смотрим, в какой позиции упорядоченного ряда он находится. $s_i$ ранг модуля $|z_i|$

###### Статистический критерий


$U = \sum I(z_i < 0) \cdot s_i$

статистика критерия принимает значения из интервала

$[0, \frac{n(n+1)}{2}]$

В случае, если верна $H_0$ значения статистики симметрично расположены относительно центра интервала.

###### Пример

$H_0: P(z_i < 0) = P(z_i > 0$

$H_1: P(z_i < 0) > P(z_i > 0$

$V_k = (c_1, \frac{n(n+1)}{2})$

значение $c_1$ можно найти с помощью таблиц критических значений определив $n$ и задав уровень значимости $\alpha$.

В случае когда объем выборки велик, мы можем воспользоваться ассимптотическими свойствами и постриоить новую статистику $\tilde U$ центрированную и нормированную.

$EU = \frac{n(n+1)}{4}$

$DU = \frac{}{}$ не сказал преподаватель

$\tilde U = \frac{U - \frac{n(n+1)}{4}}{} \xrightarrow[n \rightarrow \infty]{H_0 \; d} N(0, 1)$

Данный критерий можно использовать для проверки гипотезы, что медиана равна какому-нибудь значению

$H_0: \mu = \mu_0$

в этом случае $z_i = x_i - \mu_0$

###### Задача №1

Десяти людям была предложена специальная диета для похудания. После двухнедельного питания по этой диете масса тела участников эксперимента изменилась следующим образом:

До 69,80,92,81,70,79,78,66,57,77

После 60,84,87,79,73,71,72,67,59,70

Оказывает ли эта диета какое-либо существенное действие на массу тела? Найдите значение статистики критерия знаков. Уровень значимости 0.05.

**Решение:**

$H_0: P(z_i < 0) = P(z_i > 0) = \frac{1}{2}$

$H_1: P(z_i < 0) < P(z_i > 0)$

Необходимо построить выборку $Z = (z_1, ..., z_n)$, где $z_i = x_i - y_i$

Найти значение статистики критерия знаков

$L = \sum I(z_i < 0) \sim Bin(n, \frac{1}{2})$

In [ ]:
X = np.array([69,80,92,81,70,79,78,66,57,77])
Y = np.array([60,84,87,79,73,71,72,67,59,70])
Z = X - Y
L = np.sum(Z < 0)
L

4

Найдите значение статистики критерия Вилкоксона.

$U = \sum I(z_i < 0) \cdot s_i$, где

$s_i$ ранг, модуля статистики $Z = X - Y$




In [ ]:
U, p_value = st.wilcoxon(X, Y)
U

12.5

Оказывает ли эта диета какое-либо существенное действие на массу тела? С помощью критерия знаков и критерия Вилкоксона ответьте на вопрос. Уровень значимости 0.05.

т.к. мы рассматриваем левостороннюю альтернативную гипотезу $H_1: P(z_i < 0) < P(z_i > 0)$, критическая область у обоих критериев будет

$V_k: [0, c_{\alpha})$, где

$c_{\alpha}$ критическая точка

Критическую область для **Критерия знаков** $L = \sum I(z_i < 0) \sim Bin(n, \frac{1}{2})$. 



Найдём $c_{\alpha}$

In [ ]:
alpha = 0.05
n = 10
p0 = 0.5
c_alpha = st.binom.ppf(alpha, n, p0)
c_alpha

2.0

$V_k: [0, 2)$

$L=4$

$L \notin V_k \Rightarrow H_0$ не отвергается

Критическую область для **Критерия знаковых ранговых сумм** $U = \sum I(z_i < 0) \cdot s_i$, где

$s_i$ ранг, модуля статистики $Z = X - Y$

Найдём критическое значение $c_{\alpha} = W_{\alpha, n}$ с уровнем значимости $\alpha$ и кол-ом выборки $n$ из [таблицы](https://hostingkartinok.com/show-image.php?id=e8485e09380e2c3b479bbbaca8720f5a)

$c_{\alpha} = 11$

$V_k: [0, 11)$

$U = 12.5$

$U \notin V_k \Rightarrow H_0$ не отвергается





**Ответ:**

Критерий знаков L = 4

Критерий знаковых ранговых сумм U = 12.5

В обоих случаях $H_0$ не отвергается. Значит диета не действует!

###### Задача №2

Если смотреть на луну которая находится на горизонте, она кажется по размеру боьлше, чем луна находящаяся в зените.

В 1940 г. Халвой и Боринг предположили, что это связано с тем, что в одном случае человек смотрит на луну прямо, а в другом случае ему приходится поднимать глаза или голову. Они решили проверить эту гипотезу и провели следующий эксперимент.

В эксперименте участвовало 10 испытуемых. У них была аппаратура, которая позволяла показывать две искуственных луны, одну на уровне горизонта, другую в зените. А так же следить за тем, как смотрят на луну испытуемый, прямо или ему приходится поднимать взгляд. Каждый испытуемый смотрел на две луны. На луну находящуюся на горизонте, он смотрел прямо, а на луну в зените один раз смотрел прямо, другой раз поднимая взгляд. Испытуемые должны были сказать ощущаемые размеры луны. В итоге сравнивались значения отношений разера в зените и горизонте. Было получено 10 парных наблюдений, каждая пара соотвествует определенному испытуемому. 

на уровне X 1.65, 1.0, 2.04, 1.25, 1.05, 1.02, 1.67, 1.86, 1.56, 1.73

выше Y 1.73, 1.06, 2.03, 1.4, 0.95, 1.13, 1.41, 1.73, 1.63, 1.56





$H_0: F(x) = G(x)$ для всех $x \in \mathbb R$

$H_1: F(x) \leq G(x)$ для всех $x \in \mathbb R$

Учёные предположили, что луна больше на горизонте, потому что люди смотрят на неё прямо, если следовать этой гипотезе, тогда значения должны быть

$x_i > y_i$

Поэтому мы выбрали такую альтернативную гипотезу $H_1$.

Проверим гипотезу используя критерий знаковых ранговых сумм Вилкоксона.

$U = \sum I(z_i < 0) \cdot s_i$, где

$s_i$ ранг, модуля статистики $Z = X - Y$



In [ ]:
X = np.array([1.65, 1.0, 2.04, 1.25, 1.05, 1.02, 1.67, 1.86, 1.56, 1.73])
Y = np.array([1.73, 1.06, 2.03, 1.4, 0.95, 1.13, 1.41, 1.73, 1.63, 1.56])
# таким способом можно найти статистику U, но только у нас альтернатива левосторонняя, а тут стоит двухсторонняя. 
# ответ верный, но возможно это совпадение
U, p = st.wilcoxon(X, Y, alternative='two-sided')
U

23.0

Построим оценку вручную.

Найдём $Z = X - Y$

Нам необходимо найти модули $z_i$ и упорядочить их по возрастанию.

In [ ]:
Z = X - Y
z_rank = np.sort(abs(Z))
# отберем отрицательные значения, ранги которых нужно найти, возьмем от них модуль, т.к. в z_rank у нас модули
search = abs(Z[Z < 0])
# найдём индексы отрицательных значений, т.к. индексация идёт с 0, а нам нужны ранги с индексацией с 1, мы добавим единицу
U = (np.searchsorted(z_rank, search) + 1).sum()
U

23

Найдём критическое значение $W_{\alpha, n}$ с уровнем значимости $\alpha$ и кол-ом выборки $n$ из [таблицы](https://hostingkartinok.com/show-image.php?id=e8485e09380e2c3b479bbbaca8720f5a).

$\alpha = 0.05$

$n = 10$

$W_{\alpha, n} = 11$

Тогда критическая область для $H_1: F(x) \leq G(x)$ будет

$V_k: [0, 11)$

$U \notin V_k \Rightarrow H_0$ не отвергается.

**Ответ:**

$H_0$ не отвергается.

#### Моделирование и выводы

При моделировании было рассмотрено два подхода к проверке гипотезы, параметрические (T-test)  непараметрические (Wilcox-test).

Выводы:
1. При больших выборках работают два критерия одинаково хорошо.
2. При малых выборках и при нормальном законе распределении работает лучше T-test, на других распределениях работает хуже чем Wilcox-test.
3. При малых выборках в экспоненциально распределении, работает лучше Wilcox-test.

#### Проверка гипотез и доверительные интервалы

Рассмотрим генеральную совокупность $\xi$, предположим, что точность ф-и распределения $F(x, \theta)$ известна с точностью до параметра $\theta$.

$H_0: \theta = \theta_0$

$H_1: \theta \neq \theta_0$

Построим доверительный интервал

$(S_1, S_2): P(\theta \in (S_1, S_2)) = 1- \alpha$

Тогда

$ P(\theta_0 \in (S_1, S_2)) = 1 - \alpha$

$ P(\theta_0 \in \Theta \setminus (S_1, S_2)) = \alpha$, где

$\Theta$ - область всех возможных значений параметра $\theta$

Тогда, если

$\theta_o \in (S_1, S_2) \Rightarrow H_0$

##### Проверка равенства мат. ож.

$H_0: a_1 = a_2$

$H_1: a_1 \neq a_2$

В этом случае необходимо построить дов. интервал для параметра $a = a_1 - a_2$

Тогда проверяют

$0 \in (S_1, S_2)$

##### Проверка равенства дисперсии

$H_0: \sigma^2_1 = \sigma^2_2$

$H_1: \sigma^2_1 \neq \sigma^2_2$

В этом случае необходимо построить дов. интервал для параметра $F = \frac{\sigma^2_1}{\sigma^2_2}$

Тогда проверяют

$F \in (S_1, S_2)$